cita-executor/core/src/libexecutor/block.rs

```diff
@@ -240,7 +241,7 @@ impl ExecutedBlock {
                         Some(ReceiptError::NoCallPermission)
                     }
                     ExecutionError::Internal { .. } => Some(ReceiptError::ExecutionInternal),
-                    ExecutionError::TransactionMalformed { .. } => {
+                    ExecutionError::InvalidTransaction { .. } => {
```

Comment:
Use `ExecutionError::InvalidTransaction`, do not need `{ ..}` ?

Reply:
Done